components/Chat/Message/MessageContextMenu.tsx

```diff
@@ -1,17 +1,16 @@
-import { AnimatedBlurView } from "@components/AnimatedBlurView";
 import TableView, { TableViewItemType } from "@components/TableView/TableView";
-import { backgroundColor } from "@styles/colors";
+import { BlurView } from "@design-system/BlurView";
+import { useAppTheme } from "@theme/useAppTheme";
 import { calculateMenuHeight } from "@utils/contextMenu/calculateMenuHeight";
 import {
   AUXILIARY_VIEW_MIN_HEIGHT,
-  BACKDROP_DARK_BACKGROUND_COLOR,
-  BACKDROP_LIGHT_BACKGROUND_COLOR,
   HOLD_ITEM_TRANSFORM_DURATION,
   ITEM_WIDTH,
   OUTTER_SPACING,
   SIDE_MARGIN,
   SPRING_CONFIGURATION,
-  STATUS_BAR_HEIGHT,
+  MENU_GAP,
+  AUXILIARY_VIEW_GAP,
```

Comment:
Is there a reason we use them as const instead of having them in the theme?

Reply:
I'll migrate the constants we need to the theme

Reply:
@thierryskoda I've migrated to the theme, migration of the message list context menu to the design system is todo (this PR is about the message context menu).

Diff available here: https://github.com/ephemeraHQ/converse-app/pull/1207/files/742d758d25d91a162bac7a32c8f65693856b49bf..bbaf9ec123592d61e4898fbe5b85655d91f40a90